qiskit_experiments/library/tomography/qpt_experiment.py

```diff
@@ -52,50 +53,69 @@ class ProcessTomography(TomographyExperiment):
     def __init__(
         self,
         circuit: Union[QuantumCircuit, Instruction, BaseOperator],
+        backend: Optional[Backend] = None,
+        physical_qubits: Optional[Sequence[int]] = None,
         measurement_basis: basis.MeasurementBasis = basis.PauliMeasurementBasis(),
+        measurement_indices: Optional[Sequence[int]] = None,
         measurement_qubits: Optional[Sequence[int]] = None,
         preparation_basis: basis.PreparationBasis = basis.PauliPreparationBasis(),
+        preparation_indices: Optional[Sequence[int]] = None,
         preparation_qubits: Optional[Sequence[int]] = None,
         basis_indices: Optional[Iterable[Tuple[List[int], List[int]]]] = None,
         qubits: Optional[Sequence[int]] = None,
+        analysis: Optional[Union[BaseAnalysis, None]] = "default",
```

Comment:
The `Optional` is redundant because `Union[X,None] == Optional[X]`. Also, maybe this is an established convention in experiments, in which case ignore, but `"default"` is a `str`, not present in the type annotation.

This comment applies to all three files.

Reply:
My interpretation of Optional was it has a default value and then the type arg is the non-default value types a user is expected to supply, which in this case is BaseAnalysis and None (I guess None should actually be NoneType?). Not sure this interpretation is the correct one, though I didn't want to put string in as a type since a user shouldn't be supplying strings for the arg value